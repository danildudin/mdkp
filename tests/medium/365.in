30 8
699 403 63 69 862 88 640 212 823 677 909 368 224 364 159 787 237 201 634 819 29 501 739 659 251 796 515 993 586 856 
730 985 511 23 374 996 446 361 455 56 327 227 845 800 358 64 232 638 220 704 382 718 809 930 676 482 498 165 605 251 8362
162 991 431 85 893 242 541 865 131 648 184 662 563 415 559 980 244 659 169 748 186 654 653 472 446 345 455 583 387 867 7772
249 102 972 589 320 101 235 639 906 271 806 417 340 619 513 875 329 64 379 731 250 225 83 584 193 7 353 699 386 955 9890
735 277 852 681 263 107 994 1 88 87 836 758 444 276 999 888 903 464 287 796 151 57 842 603 543 703 781 767 10 927 8010
347 670 763 831 953 309 813 573 25 188 902 886 667 357 357 645 900 684 520 759 666 424 806 933 765 251 48 839 400 56 5168
495 320 713 851 408 452 251 546 922 80 760 483 903 47 350 24 471 855 78 911 66 321 987 677 742 482 404 659 122 511 6580
125 965 338 516 290 434 628 542 507 709 874 61 639 356 714 990 539 497 981 362 571 767 616 695 450 336 776 963 174 87 7117
219 251 797 329 320 208 751 536 324 104 934 944 235 540 235 697 861 568 861 686 147 133 894 736 341 147 800 258 280 247 6004
