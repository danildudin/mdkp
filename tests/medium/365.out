9172
13
0 8 9 10 13 18 19 21 25 26 27 28 29 
